styles/cspace/SortSelect.css

```diff
@@ -17,3 +17,7 @@
 .common:focus {
   outline: 2px solid highlightColor;
 }
+
+.commonLabel {
```

Comment:
The class name can just be `label`, but I don't think this is needed. Try using a space character in the JSX to get the appropriate spacing.